contrib/epee/include/net/http_server_handlers_map2.h

```diff
@@ -71,7 +71,7 @@
       MINFO(m_conn_context << "calling " << s_pattern); \
       if(!callback_f(static_cast<command_type::request&>(req), static_cast<command_type::response&>(resp), &m_conn_context)) \
       { \
-        LOG_ERROR("Failed to " << #callback_f << "()"); \
+        LOG_ERROR("Failed to " << #callback_f << "() for Host " << m_conn_context.m_remote_address.host_str()); \
```

Comment:
Pedantic, but can you do it like the MINFO three lines above ?

Reply:
I didn't want to change the existing message text, in case anything else was already parsing it.
Are you suggesting `MERROR(m_conn_context << "failed to " #callback_f << "()");` here?

Reply:
Yes. Easier when all the messages have the IP in the same format at start.